src/cdk/testing/webdriver/webdriver-element.ts

```diff
@@ -0,0 +1,229 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  _getTextWithExcludedElements,
+  ElementDimensions,
+  EventData,
+  ModifierKeys,
+  TestElement,
+  TestKey,
+  TextOptions
+} from '@angular/cdk/testing';
+import * as webdriver from 'selenium-webdriver';
+import {getWebDriverModifierKeys, webDriverKeyMap} from './webdriver-keys';
+
+/** A `TestElement` implementation for WebDriver. */
+export class WebDriverElement implements TestElement {
+  constructor(
+      readonly element: () => webdriver.WebElement,
+      private _stabilize: () => Promise<void>) {}
+
+  async blur(): Promise<void> {
```

Comment:
Do our API docs take the description from `TestElement`, or do we need to duplicate them here?

Reply:
It looks like it does not grab the description. This is an issue for the other environments too so I'll fix it in a follow-up